Files/Strings/en-US/Resources.resw

```diff
@@ -2767,4 +2767,7 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="SideBarFavoritesMoveToTop" xml:space="preserve">
     <value>Move to top</value>
   </data>
+  <data name="DetailsViewHeaderFlyout_ShowDateCreated.Text" xml:space="preserve">
+    <value>Date created column</value>
+  </data>
```

Comment:
Your code is good, but I think we can use this PR to add the other remaining string on the same flyout:

```suggestion
  </data>
  <data name="DetailsViewHeaderFlyout_ShowItemSize.Text" xml:space="preserve">
    <value>Item size column</value>
  </data>
```